ship.config.js

```diff
@@ -7,5 +7,5 @@ module.exports = {
     return true;
   },
   buildCommand: () => null,
-  afterPublish: ({ exec }) => exec('yarn run release-templates'),
```

Comment:
Did you try to rerun the job with SSH on CircleCI and execute the command above or the underlying one?
`git clone -b templates --single-branch https://github.com/algolia/create-instantsearch-app.git build`

Reply:
Oh TIL! Thanks, I'll try that, did not know we could have an interactive TTY.

Reply:
This command and `yarn release-templates` correctly work there. I'll keep looking into shipjs then.